Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -146,17 +158,18 @@ protected override void OnApplyTemplate()
         {
             if (_contentGrid != null)
             {
-                _contentGrid.ManipulationStarted -= ContentGrid_ManipulationStarted;
                 _contentGrid.ManipulationDelta -= ContentGrid_ManipulationDelta;
-                _contentGrid.ManipulationCompleted -= ContentGrid_ManipulationCompleted;
+                _contentGrid.ManipulationCompleted += ContentGrid_ManipulationCompleted;
```

Comment:
Don't know how that line got there. Looks like something is wrong with this file because I don't have any line like that in my repo. Conflict maybe?